code/game/gamemodes/cult/runes.dm

```diff
@@ -940,7 +940,7 @@ structure_check() searches for nearby cultist structures required for the invoca
 			continue
 		if(!HAS_TRAIT(O, TRAIT_RESPAWNABLE) || QDELETED(src) || QDELETED(O))
 			continue
-		if(O.mind.current && HAS_TRAIT(O.mind.current, SCRYING))
+		if(O?.mind?.current && HAS_TRAIT(O.mind.current, SCRYING))
```

Comment:
This means that the mindless ghost will be added to the rune.
```suggestion
		if(!O.mind)
				continue
		if(O.mind.current && HAS_TRAIT(O.mind.current, SCRYING))
```

Reply:
fuck